packages/coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -131,11 +145,12 @@ export function verifyCommonBundle(
 function validatePractitionerRole(
   bundle: fhir.Bundle,
   practitionerRole: fhir.PractitionerRole,
+  isResponsibleParty: boolean,
   incorrectValueErrors: Array<fhir.OperationOutcomeIssue>,
   accessTokenSDSUserID: string,
   accessTokenSDSRoleID: string
-): void{
-  if(practitionerRole.practitioner) {
+): void {
+  if (practitionerRole.practitioner && !isResponsibleParty) {
```

Comment:
We should validate that it's either a reference or an identifier if it's a responsible party

Reply:
Updated in eb325ec81a6249a7534ccc70c375f200f43f0128